cpu/esp8266/Makefile.dep

```diff
@@ -22,6 +22,10 @@ endif
 ifneq (, $(filter esp_wifi, $(USEMODULE)))
     CFLAGS += -DLWIP_OPEN_SRC
     LINKFLAGS += -Wl,-wrap=ethernet_input
+
+    # thin archives trigger a reboot loop - see #12258
+    ARFLAGS = rcs
```

Comment:
Is it really necessary for ESP8266? @LordTy said that the reboot loop does not happen for ESP8266, at least not for the `esp_now` module.

Reply:
I tried it with `esp8266-esp-12x` and `esp_wifi` and it would reboot loop there.